libretro-common/cdrom/cdrom.c

```diff
@@ -1428,7 +1431,7 @@ struct string_list* cdrom_get_available_drives(void)
    for (i = 0; i < sizeof(DWORD) * 8; i++)
    {
       char path[] = {"a:\\"};
-      char cdrom_path[] = {"cdrom://a:/drive-track01.bin"};
+      char cdrom_path[] = {"cdrom://a:/drive.cue"};
```

Comment:
just opening the drive creates the file handle needed for `cdrom_get_inquiry` with less overhead than opening a track